Firestore/Source/Public/FIRQuery.h

```diff
@@ -273,6 +273,70 @@ addSnapshotListenerWithIncludeMetadataChanges:(BOOL)includeMetadataChanges
                     arrayContains:(id)value NS_SWIFT_NAME(whereField(_:arrayContains:));
 // clang-format on
 
+/**
+ * Creates and returns a new `FIRQuery` with the additional filter that documents must contain
+ * the specified field, the value must be an array, and that array must contain at least one value
+ * from the provided array.
+ *
+ * A query can have only one arrayContains filter and it cannot be combined with arrayContains or
+ * in.
+ *
+ * @param field The name of the field containing an array to search.
+ * @param value The value that contains the values to match.
+ *
+ * @return The created `FIRQuery`.
+ */
+// clang-format off
+- (FIRQuery *)queryWhereField:(NSString *)field arrayContainsAny:(id)value;
```

Comment:
This is missing an `NS_SWIFT_NAME` annotation so I suspect this API does not look right in swift. Look at how `-queryWhereField:arrayContains:` is annotated to get an idea for how this should look.

Also I can't find an entry in `BasicCompileTests.swift` that verifies this looks right in Swift. All public Objective-C APIs need this because we're relying on the Swift compiler to do this translation for us, but the default isn't always right.